docs/contributing/how-to-contribute-to-docs.md

```diff
@@ -63,11 +63,26 @@ Keep `menu` the same as sub-directory the file is in. This will help to manage a
 
 Show new menu section in main page by changing `website/layouts/_default/baseof.html` file.
 
+## Logos
```

Comment:
Isn't that too hidden? Just curious how we can make it more searchable. Maybe some hover over logos "Go there " etc?

Reply:
Ok, let's ignore my comment. It's fine to this being there we can just redirect people if they will aks to be on page (: